packages/flutter_tools/lib/src/platform_plugins.dart

```diff
@@ -132,15 +135,7 @@ class AndroidPlugin extends PluginPlatform {
       );
     }
 
-    String mainClassContent;
```

Comment:
This is not necessary because the error_handling_io.dart implementation of FileSystem will catch permission errors